pkg/queue/local_queue.go

```diff
@@ -23,11 +23,17 @@ import (
 	"sigs.k8s.io/kueue/pkg/workload"
 )
 
+const DefaultLocalQueueName = "default"
```

Comment:
The name seems a bit sparse for my taste. Could we consider something like defaultQueue?

Reply:
Also can we define this [here](https://github.com/kubernetes-sigs/kueue/blob/aefe6a94800e76b71bb0a157143653a2199a8bba/pkg/constants/constants.go)?

Reply:
The queue still should be manually created by administrator, so I picked the most simplest name to minimise chances of typos. 
From the user point of view they will see "kueue.x-k8s.io/queue-name:default" annotation.